util/src/main/java/io/kubernetes/client/apimachinery/GroupVersion.java

```diff
@@ -12,24 +12,24 @@
 */
 package io.kubernetes.client.apimachinery;
 
+import static io.kubernetes.client.util.Preconditions.precondition;
+
 import io.kubernetes.client.common.KubernetesObject;
 import io.kubernetes.client.util.Strings;
 import java.util.Objects;
 
 public class GroupVersion {
 
+  private static final GroupVersion GROUP_VERSION_V1 = new GroupVersion("", "v1");
```

Comment:
5. it is an immutable class, for `v1` let's return the same instance, instead of creating a new one.